Clicker/ViewControllers/LoginViewController.swift

```diff
@@ -12,21 +12,51 @@ import SnapKit
 
 class LoginViewController: UIViewController, GIDSignInUIDelegate {
     
-    var appLogoImage: UIImageView!
+    var gradient: CAGradientLayer!
     var signInButton: GIDSignInButton!
+    var welcomeLabel: UILabel!
+    var appNameLabel: UILabel!
+    var pronunciationLabel: UILabel!
     
     // MARK: - INITIALIZATION
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .clickerGreen0
         
-        GIDSignIn.sharedInstance().uiDelegate = self
+        gradient = CAGradientLayer()
+        gradient.colors = [UIColor.bluishGreen.cgColor, UIColor.aquaMarine.cgColor]
+        gradient.startPoint = CGPoint(x: 1.0, y: 1.0)
+        gradient.endPoint = CGPoint(x: 0.0, y: 0.0)
+        gradient.frame = view.bounds
+        view.layer.addSublayer(gradient)
+        
+        welcomeLabel = UILabel()
+        welcomeLabel.text = "Welcome to"
```

Comment:
can you put the string as a constant class variable (under the var class variables), i.e.:
`let welcomeLabelText = "Welcome to"`